From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ronald Claveau <linux-kernel-dev@aliel.fr>
Date: Mon, 4 May 2026 16:57:36 +0200
Subject: dt-bindings: phy: amlogic: Add compatible for T7

The T7 SoC integrates a PCIe PHY that shares its register layout and
initialization sequence with the existing g12a USB3/PCIe combo PHY.

Add T7 SOC PCIe compatible which fallback to G12A one.

Signed-off-by: Ronald Claveau <linux-kernel-dev@aliel.fr>
Signed-off-by: Nick Xie <nick@khadas.com>
Change-Id: I07bdde6338404d730bf52799a1188e41455ccd64
---
 Documentation/devicetree/bindings/phy/amlogic,g12a-usb3-pcie-phy.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/phy/amlogic,g12a-usb3-pcie-phy.yaml b/Documentation/devicetree/bindings/phy/amlogic,g12a-usb3-pcie-phy.yaml
index 111111111111..222222222222 100644
--- a/Documentation/devicetree/bindings/phy/amlogic,g12a-usb3-pcie-phy.yaml
+++ b/Documentation/devicetree/bindings/phy/amlogic,g12a-usb3-pcie-phy.yaml
@@ -16,6 +16,7 @@ properties:
       - items:
           - enum:
               - amlogic,t7-usb3-phy
+              - amlogic,t7-pcie-phy
           - const: amlogic,g12a-usb3-pcie-phy
       - enum:
           - amlogic,g12a-usb3-pcie-phy
-- 
Armbian

